src/Controls/src/Core/Platform/AlertManager/AlertManager.Windows.cs

```diff
@@ -70,6 +70,10 @@ void OnPageBusy(Page sender, bool enabled)
 
 			async void OnAlertRequested(Page sender, AlertArguments arguments)
 			{
+				// Verify that the page making the request is part of the current Window. 
+				if (!PageIsInThisWindow(sender))
+					return;
```

Comment:
@PureWeen  maybe this is a sign that the alert manager should be scoped? I thought it was, so maybe this is another bug?